nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/KeyDerivationFunction.java

```diff
@@ -44,6 +47,10 @@ public String getDescription() {
         return description;
     }
 
+    public boolean isStrongKDF() {
+        return (name.equalsIgnoreCase(BCRYPT.name) || name.equals(SCRYPT.name) || name.equals(PBKDF2.name));
```

Comment:
is there a reason for checking equalsIgnoreCase for BCRYPT but just equals for SCRIPT, PBKDF2?